back/engines/free/email_campaigns/app/views/email_campaigns/internal_comment_on_idea_assigned_to_you_mailer/campaign_mail.mjml

```diff
@@ -0,0 +1,3 @@
+<%= render partial: 'email_campaigns/internal_comments/internal_comment_mail_body', locals: {internal_comment_body: event.internal_comment_body} %>
+<%= render partial: 'email_campaigns/posts/post_with_image', locals: {post_image_url: event&.post_image_medium_url, post_title_multiloc: event.post_title_multiloc, post_body_multiloc: event.post_body_multiloc} %>
```

Comment:
It looks like these 3 lines are repeated among all new campaigns. Can they be put in a partial?